recipes/muparser/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.2.5" %}
+
+package:
+  name: muparser
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/beltoforion/muparser/archive/v{{ version }}.tar.gz 
+  md5: 02dae671aa5ad955fdcbcd3fee313fb7
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - gcc  # [linux]
+  run:
+    - libgcc  # [linux]
+
+test:
+  commands:
+    - ls -l  # [unix]
```

Comment:
Can you add a real test? Sometimes just checking is an expected library exists is enough. See https://github.com/conda-forge/hdfeos2-feedstock/blob/master/recipe/meta.yaml#L24-28 for an example.